lib/effect/DisplayColorPaletteEffect.cpp

```diff
@@ -5,10 +5,16 @@ DisplayColorPaletteEffect::DisplayColorPaletteEffect() : Effect() {}
 CRGB DisplayColorPaletteEffect::GetRGB(uint8_t led_index, uint32_t time_ms,
                                        const StripDescription &strip,
                                        RadioPacket *setEffectPacket) {
-  UNUSED(led_index);
   ColorPalette palette =
       palettes()[setEffectPacket->readPaletteIndexFromSetEffect()];
-  CHSV color = palette.GetGradient((time_ms / 2) * 23);
+  CHSV color;
+  if (strip.led_count < palette.Size() && palette.Size() <= 4) {
```

Comment:
What is this fixing? Overflow?

Reply:
Before this, if the palette was larger than the number of LEDs, it wouldn't display correctly.